templates/repo/diff/conversation.tmpl

```diff
@@ -1,9 +1,12 @@
-{{$resolved := (index .comments 0).IsResolved}}
-{{$invalid := (index .comments 0).Invalidated}}
-{{$resolveDoer := (index .comments 0).ResolveDoer}}
-{{$isNotPending := (not (eq (index .comments 0).Review.Type 0))}}
-{{$referenceUrl := printf "%s#%s" $.Issue.Link (index .comments 0).HashTag}}
-<div class="conversation-holder" data-path="{{(index .comments 0).TreePath}}" data-side="{{if lt (index .comments 0).Line 0}}left{{else}}right{{end}}" data-idx="{{(index .comments 0).UnsignedLine}}">
+{{$comment := index .comments 0}}
+{{if $comment}}
```

Comment:
Maybe indent below after review is done.

Reply:
Then it won't backport easily.

I will reformat the code after merge.

Reply:
Reformated now, with a force push